app/assets/stylesheets/application.scss

```diff
@@ -16,6 +17,7 @@ $govuk-assets-path: "/";
 // If you absolutely need to override gov.uk styles, don't,
 // instead create new classes or ids with a prefix based on the app name.
 
+@import "accessible-autocomplete";
```

Comment:
To discuss: introduction if a scalable and maintainable CSS architecture as we are writing additional app specific styles.

Reply:
Happy to go with a css architecture. I don't have loads of experience so if you want to recommend one, I'm happy to go with it. Also happy to a have a further discussion too

Reply:
Ticket created to add css architecture: https://dsdmoj.atlassian.net/browse/AP-5595